.github/workflows/ci.yml

```diff
@@ -11,7 +11,6 @@ jobs:
       fail-fast: false
       matrix:
         java-version:
-          - 11
```

Comment:
this should come before `Migrate units validation to jakarta` commit

Reply:
Rearranged